### PR TITLE
add github workflows

### PR DESCRIPTION
**Pull request by nairbv**
just copied from fms and changed target, should basically be the same here.

### CODE REVIEW

**Review by JRosenkranz — APPROVED** [review]
lgtm

File: .github/workflows/mypy.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        pip install transformers==4.35.0
+        pip install types-requests
+        pip install mypy
+        pip install pyarrow-stubs==10.0.1.7
+        pip install safetensors
+        pip install .
+        
+    - name: Test with pytest
+      run: |
+        mypy --exclude hf --exclude testing fms-extras
```
Comment (JRosenkranz) [inline .github/workflows/mypy.yml]: should this be `fms_extras`

**Review by JRosenkranz — COMMENTED** [review]
small comment regarding name

**Comment by nairbv** [comment]
merging despite test failures. failure error is `5` meaning no tests found. it'll work on any new branch adding tests (and the next thing to land will have tests)

**Pull request merged**